README.md

```diff
@@ -560,8 +560,8 @@ yargs.command('get <source> [proxy]', 'make a get HTTP request', require('my-mod
   .argv
 ```
 
-.completion(cmd, [description], [fn]);
--------------
+.completion([cmd, [description], [fn]]);
```

Comment:
slight typo, I would probably write this like `.completion([cmd], [description], [fn]);`